backend/vaa-strapi/src/extensions/users-permissions/controllers/candidate.ts

```diff
@@ -49,7 +49,12 @@ module.exports = {
     }
 
     return {
-      candidate: await sanitizeCandidate(candidate, ctx)
+      candidate: {
+        ...(await sanitizeCandidate(candidate, ctx)),
+        //Return email associated with the registration key so that the login
+        //page can be prefilled on navigation
+        email: candidate.email
+      }
```

Comment:
Since the sanitizeCandidate function isn't used elsewhere, I suppose the email could be added to the response there instead of here.